testhelper/integrationtesthelper.go

```diff
@@ -281,15 +277,18 @@ func IntegrationCartographer(
 	} else {
 		attrCfg.Camera = map[string]string{
 			"name":              string(LidarWithErroringFunctions),
-			"data_frequency_hz": strconv.Itoa(defaultLidarTimeInterval),
+			"data_frequency_hz": "200",
```

Comment:
[opt] Defining 200 here and in consts may be less beneficial then just defining the const as 200 and being able to use it in both places in my opinion.

Reply:
change to 5